charts/prow/templates/ingress.yaml

```diff
@@ -0,0 +1,41 @@
+# Copyright 2016 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: extensions/v1beta1
+kind: Ingress
+metadata:
+  name: '{{ template "fullname" . }}'
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "prow-ingress"
```

Comment:
Can we document in the README that an IP with the name 'prow-ingress' should already exist?